test/integration/lke_clusters_test.go

```diff
@@ -74,8 +74,8 @@ func TestLKECluster_Update(t *testing.T) {
 
 	updatedTags := []string{"test=true"}
 	updatedLabel := cluster.Label + "-updated"
-	updatedK8sVersion := "1.23"
-	isHA := true
+	updatedK8sVersion := "1.30"
+	isHA := false
```

Comment:
Note for other reviewers:

As of now HA cannot be specified in the same PUT request as a K8s version upgrade, which caused issues here because we can't guarantee there will be multiple LKE versions available at a time.